src/v0/destinations/freshsales/data/SalesActivityConfig.json

```diff
@@ -57,7 +57,6 @@
   {
     "destKey": "created_at",
     "sourceKeys": "originalTimestamp",
```

Comment:
why not refer `timestamp` here ?

Reply:
Do we know what's expected from destination regarding these 2 fields `created_at` & `updated_at` ?
Since they are timestamp datatype, assigning any random timestamp field is not right. In the worst case, both should be set to `timestamp` field